src/catkin_pkg/terminal_color.py

```diff
@@ -42,26 +42,25 @@
 
 def ansi(key):
     """Return the escape sequence for a given ansi color key."""
-    global _ansi
     return _ansi[key]
 
 
 def enable_ANSI_colors():
     """Populate the global module dictionary `ansi` with ANSI escape sequences."""
-    global _ansi
```

Comment:
This is in the commit history now but as it happens this usage was *not* spurious nor flagged by flake8 but I assumed that 2/3 == 3/3 and removed it anyway. In order to fix the failing test without restoring the use of the global declaration I changed the code to update rather than assign and I think it's cleaner.